inductiva/structures/scenarios/plate_linear_elastic/mesh_utils.py

```diff
@@ -0,0 +1,257 @@
+"""Utils to create the mesh."""
+
+from typing import List
+from typing import Optional
+
+import gmsh
+
+import geometry_utils
+
+
+class Mesh:
+    """Mesh.
+
+    Attributes:
+        geometry: A geometry_utils.GeometricCase object.
+        glb_ref_fac (float): The refinement factor for global refinement of the
+          mesh. 
+        loc_ref_fac (float): The refinement factor for local refinement of  the
+          mesh.
+    """
+
+    def __init__(self,
+                 geometry: geometry_utils.GeometricCase,
+                 glb_ref_fac: Optional[float] = 1.0,
+                 loc_ref_fac: Optional[float] = 0.0) -> None:
```

Comment:
Be more expressive in the names of these attributes, `global_refinement_factor`, `local_refinement_factor`. I know they are lengthy, but that's fine. They are clear too!